lte/protos/mconfig/mconfigs.proto

```diff
@@ -71,6 +71,14 @@ message EnodebD {
         int32 tac = 8;
         int32 cell_id = 9;
         string ip_address = 10;
+        PowerControl power_control = 34;
+        bool x2_enable_disable = 112;
```

Comment:
can you please use sequential numbers

Reply:
In order to solve the conflict between parameters supported by different versions and coding by different people, we specify the number submitted by each person. If it must modify, I will modify at present. 
![number](https://user-images.githubusercontent.com/86106068/133041530-16ab1159-2823-492b-8e46-cf5ee8a524d8.JPG)